src/templates/ucp.html

```diff
@@ -380,7 +376,7 @@ <h3>{{ 'Experiment Templates'|trans }}</h3>
             <i data-toggle='dropdown' aria-haspopup='true' aria-expanded='false' class='dropdown-toggle fas fa-ellipsis-h fa-2x fa-pull-right clickable' title='{{ 'More options'|trans }}' aria-label='{{ 'More options'|trans }}' /></i>
             <div class='dropdown-menu' aria-label='{{ 'More options'|trans }}'>
             <!-- MANAGE PERMISSIONS -->
-            <a class='dropdown-item modalToggle' data-modal='permModal' href='#'><i class='fas fa-share-alt' title='{{ 'Manage Permissions'|trans }}' /></i> {{ 'Manage Permissions'|trans }}</a>
+            <a class='dropdown-item modalToggle' data-modal='permModal' href='#' data-rw="{{ template.canread }}" data-wr="{{ template.canwrite }}"><i class='fas fa-share-alt' title='{{ 'Manage Permissions'|trans }}' /></i> {{ 'Manage Permissions'|trans }}</a>
```

Comment:
data-read and data-write would be better names, as rw and wr can easily be mistaken.

Reply:
Has been renamed.